cluster/terraform_kubernetes/config/prometheus/development.filebeat.yml.tmpl

```diff
@@ -0,0 +1,17 @@
+filebeat.autodiscover:
+  providers:
+      - type: kubernetes
+        templates:
+          - condition:
+              equals:
+                kubernetes.annotations.logit.io/send: "true"
+            config:
+              - type: container
+                paths:
+                  - "/var/log/containers/*-$${data.kubernetes.container.id}.log"
+                json.keys_under_root: true
```

Comment:
Logit.io support recommended parsing json on the logstash side. Let's remove this line and configure the logstash pipeline.